datahub-graphql-core/src/main/resources/gms.graphql

```diff
@@ -108,6 +108,9 @@ type Query {
 
 type Mutation {
     updateDataset(input: DatasetUpdateInput!): Dataset
+    updateChart(input: ChartUpdateInput!): Chart
+    updateDashboard(input: DashboardUpdateInput!): Dashboard
+    updateTag(input: TagUpdate!): Tag
```

Comment:
Nit: For consistency, the input type should be TagUpdateInput

Reply:
So, its a bit tricky bc XYZUpdateInput is the name for input methods and XYZUpdate is the name for subtypes. TagUpdate is a subtype and a main input, so its not quite possibly to be consistent with both. This to me made more sense so it didn't confuse anyone when used as a subtype